### PR TITLE
fix docs for lr

### LINKED ISSUE #27730: [docs] reorder docs for torch.optim.lr_scheduler.CyclicLR
## 📚 Documentation

Here: https://pytorch.org/docs/master/optim.html?highlight=cycliclr#torch.optim.lr_scheduler.CyclicLR

image

the three built-in policies can be illustrated in the Parameter section.

cc @vincentqb

### PR DESCRIPTION
**Pull request by vincentqb**
Documentation for learning rate does not render well. Fixes #27730.

Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#28026 fix docs for lr**

Differential Revision: [D17953395](https://our.internmc.facebook.com/intern/diff/D17953395)

### CODE REVIEW

**Comment by zou3519** [comment]
Could you build the docs locally (via `cd docs && make html && open build/html`)and attach a screenshot of the updated page?

**Comment by vincentqb** [comment]
Here's the output.

Screen Shot 2019-10-16 at 11 04 34 AM

Screen Shot 2019-10-16 at 11 09 42 AM

Unfortunately, there's not as much white space after the enumerated list as I would like. It's still much better than what was there before :)

**Review by zou3519 — APPROVED** [review]
I'm not sure what to do about the space, but this lgtm
